deepinv/__init__.py

```diff
@@ -55,17 +55,7 @@
 
 __all__ += ["unfolded"]
 
-from deepinv.training_utils import train, test
+from deepinv.training_utils import train, test, Trainer

```

Comment:
Can we also put this in so we can import other files from `training_utils`?
```python
from deepinv import training_utils
__all__ += ["training_utils"]
```

Reply:
mm I'm not sure about this, if we have very specific functions in the training_utils, we don't want to be able to import them directly as deepinv.function, but rather as deepinv.training_utils.function

Reply:
Sorry that was a problem with my linter. Regardless, doing this still shouldn't allow `import deepinv.function`, and including this is for completeness more than anything

Reply:
done